main.go

```diff
@@ -32,7 +32,7 @@ func main() {
 			"version": package_version,
 		})
 
-	logger := log.NewLoggerWrapper(commands.Verbose)
+	logger := log.NewLoggerWrapper(commands.Verbose, commands.LogLevel)
```

Comment:
The two-parameter solution seems overly complicated - it would probably make more sense to handle the `verbose` parameter here and set the log level to "debug", then it's just `main` that has to worry about it.